### PR TITLE
chore: Mattermost message after publication

### PR DESCRIPTION
**Pull request by ptbrowne**
Post message to mattermost after publication.

- Configured MATTERMOST_HOOK_URL
- Configured MATTERMOST_CHANNEL (publication channel only for beta/stable versions)
- Updated cozy-app-publish (on other repo it is automatically updated since it is
  installed at publish time, should we do the same here ?)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,10 +10,13 @@ branches:
   - /^\d+\.\d+\.\d+(\-beta.\d+)?$/
 env:
   global:
+    - MATTERMOST_CHANNEL: "{"dev":"app---home","beta":"app---home,publication","stable":"app---home,publication"}"
```
Comment (Crash--) [inline .travis.yml]: app----home ? Publier sur feat---password-mgr ça fait peut-être plus sens ?

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   "devDependencies": {
     "@testing-library/react": "^9.4.0",
     "babel-preset-cozy-app": "1.7.0",
-    "cozy-app-publish": "0.22.1",
+    "cozy-app-publish": "^0.22.3",
```
Comment (Crash--) [inline package.json]: On devrait le passer sur le travis yaml comme les autres

File: .travis.yml (hunk 1 above)
Comment (ptbrowne) [reply .travis.yml]: Ah oui je me suis trompé.

File: package.json (hunk 2 above)
Comment (ptbrowne) [reply package.json]: Done

File: .travis.yml (hunk 1 above)
Comment (ptbrowne) [reply .travis.yml]: Done.

**Pull request merged**